translations/zh_CN.ts

```diff
@@ -2161,8 +2161,8 @@ Output: <byte value="xd"/>
         <translation>检测到 Deepin</translation>
     </message>
     <message>
-        <source>Deepin plays smart and sets you the wrong HTTPS_PROXY, FTP_PROXY environment variable.</source>
-        <translation>Deepin可能自作聪明，为你设置了错误的 HTTPS_PROXY 和 FTP_PROXY 环境变量。</translation>
+        <source>Deepin may sets you the wrong HTTPS_PROXY, FTP_PROXY environment variable.You can add proxy info to /home/<username>/.bashrc, such as: export https_proxy="http://127.0.0.1:8080".</source>
```

Comment:
You can see there's some grammar error in your modification.

> Deepin may **sets** you

Shall it be:

> Deepin may have set you?

Reply:
Also, you are not using the correct English / Chinese-English paragraph style. Multiple spaces are missing.